src/scss/includes/map_theme.scss

```diff
@@ -214,8 +270,13 @@
     }
   }
 
-  .mapboxgl-ctrl-bottom-right .mapboxgl-ctrl, .mapboxgl-ctrl.mapboxgl-ctrl-attrib {
+  .mapboxgl-ctrl-bottom-right .mapboxgl-ctrl {
     float: inherit;
+    margin-right: 10px;
+  }
+
+  .mapboxgl-ctrl-bottom-right  {
+    height: 55px;
```

Comment:
Is this really required? This doesn't seem to change much. Our Mapbox control CSS overrides are already a mess, far too complicated, far too many lines for so little user value, let's try no to add anything new.